Symfony/CS/Fixer/Symfony/NoEmptyCommentFixer.php

```diff
@@ -41,7 +41,7 @@ public function fix(\SplFileInfo $file, $content)
      */
     public function getDescription()
     {
-        return 'There should not be an empty comments.';
+        return 'There should not be an empty comment.';
```

Comment:
please revert (together with the README), the fixer will still fix all comments that are applicable